fvm/fvm.go

```diff
@@ -242,3 +243,74 @@ func (vm *VirtualMachine) GetAccount(
 	}
 	return account, nil
 }
+
+// GetAccountBalance returns an account balance by address or an error if none exists.
+func GetAccountBalance(
```

Comment:
Can you extract the common parts of GetAccountKeys, GetAccountBalance, GetAccount there is a lot of repeated boilerplate.